sklearn/decomposition/tests/test_kernel_pca.py

```diff
@@ -52,10 +52,22 @@ def histogram(x, y, **kwargs):
                 assert_equal(X_pred2.shape, X_pred.shape)
 
 
-def test_invalid_parameters():
+def test_kernel_pca_invalid_parameters():
     assert_raises(ValueError, KernelPCA, 10, fit_inverse_transform=True,
                   kernel='precomputed')
 
```

Comment:
pep8, please add another blank line here.

In general you should run e.g., flake8 on your code (or configure your editor to do so automatically)